ios/FluentUI.Demo/FluentUI.Demo/Demos/BottomSheetDemoController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+import FluentUI
+
+class BottomSheetDemoController: DemoController {
+// MARK: Lifecycle methods
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        setupMainPersonaListView()
+        setupBottomSheet()
+    }
+
+// MARK: Setup Methods
+
+    private func setupMainPersonaListView() {
+        view.addSubview(personaListView)
+
+        NSLayoutConstraint.activate([
+            personaListView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            personaListView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            personaListView.topAnchor.constraint(equalTo: view.topAnchor),
+            personaListView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+
+    private func setupBottomSheet() {
```

Comment:
can you add more options so we can show case expand state is full screen vs half screen(preferred content size) similar to drawercontroller demo case?

Reply:
I think it's likely that the CommandingBottomSheet will end up being the control used by clients, and the empty one might not even have a demo controller, or it will just be one option in the controller. That's why I didn't want to spend too much time on the demo for now, so I can get to the Commanding sheet asap. Is it okay if I leave this until later?

Reply:
If there's a concern with having WIP demos in the main branch, I can leave out the demo entirely until things are more polished.